### PR TITLE
Remove private internet access hooks

### PR DESCRIPTION
**Pull request by mrose17**
until licensing issues resolved.

auditor: @diracdeltas

### CODE REVIEW

File: docs/state.md [hunk 1]
```diff
@@ -384,14 +384,38 @@ WindowStore
   ledgerInfo: {
     creating: boolean,
     created: boolean,
-    reconcileStamp: number,
-    reconcileDelay: ?,
-    delayStamp: ?,
-    transactions: Array,
+    delayStamp: timestamp,
```
Comment (diracdeltas) [inline docs/state.md]: please use a js builtin type, not `timestamp`. see http://usejsdoc.org/tags-type.html

File: docs/state.md [hunk 2]
```diff
@@ -384,14 +384,38 @@ WindowStore
...
+      submissionStamp: timestamp,
+      submissionId: string,
+      count: number,
+      satoshis: number,
+      votes: number,
+      ballots: {
+        publisher1: number
+        ...
+      }
+    } ... ]
```
Comment (diracdeltas) [inline docs/state.md]: thanks for documenting these

**Comment by mrose17** [comment]
@diracdeltas - please do not commit, pending finalization, etc.

**Comment by diracdeltas** [comment]
closing in light of recent emails

**Pull request closed without merging**